yorkie/src/main/kotlin/dev/yorkie/core/PresenceProxy.kt

```diff
@@ -0,0 +1,32 @@
+package dev.yorkie.core
+
+import dev.yorkie.document.change.ChangeContext
+import dev.yorkie.document.time.ActorID
+
+/**
+ * [PresenceProxy] represents a proxy for the Presence to be manipulated from the outside.
+ */
+public data class PresenceProxy internal constructor(
+    private val changeContext: ChangeContext,
+    private val presences: Presences,
+    private val actorID: ActorID,
+) {
+
+    public fun set(data: Map<String, String>) {
```

Comment:
`set` could be a little misleading as it may imply that previous presence data would be overriden by new data map.
Maybe we should use `put` instead, or we could change the parameter type to collection of pairs of strings.

Reply:
I changed the function name as `put` in [this commit](https://github.com/yorkie-team/yorkie-android-sdk/pull/130/commits/77ab8e1ff2074a08645ed0359f2f98116761f759#diff-5cb2a1100515c19243e7fa5708c7f146e511e17229a2057b7dc2febbc1ab092a).